CHANGELOG.md

```diff
@@ -22,20 +21,24 @@ Allowed Types of change: `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `
 
 ### Removed
 
+## [23.5.0] - 2020-06-15
```

Comment:
Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045776)